docs/topics/proc_creating-an-application.adoc

```diff
@@ -0,0 +1,146 @@
+[#creating-an-application]
+= Creating an application
+
```

Comment:
I think this chapter could use an introduction if we wanted to it really properly, but I won't hold this review on it.

Reply:
Thank you, I think including an introduction here is beneficial. I'll add one here as well as in the deployment section.